src/helpers/getAllEvents.ts

```diff
@@ -0,0 +1,41 @@
+import { QUERY_BLOCK_LIMIT } from '@big-whale-labs/constants'
+import { SealCredLedger } from '@big-whale-labs/seal-cred-ledger-contract'
+
+export default async function (sealCredLedger: SealCredLedger) {
+  const eventsFilter = sealCredLedger.filters.SetMerkleRoot()
+  const deleteEventsFilter = sealCredLedger.filters.DeleteMerkleRoot()
+
+  let deleteTopic, setTopic
+  const topics = []
+
+  if (deleteEventsFilter.topics) {
+    deleteTopic = Array.isArray(deleteEventsFilter.topics[0])
```

Comment:
Don't spawn more terms here. Use `set merkle root` and `delete merkle root` consistently, don't do just `eventsFilter` or `deleteTopic`